web_client/analysis-modules/feature-matching.jsx

```diff
@@ -117,7 +117,115 @@ const computeMatch = (data, page) => {
   );
 };
 
-const applyMatch = (data, page) => { /* TODO(opadron): */ };
+const applyMatch = (data, page) => {
+  const truncatePromise = Promise.all([
+    D(actions.truncateAnalysisPages(1, {
+      clear: true,
+      disable: true,
+      remove: false
+    })),
+
+    (
+      applyResultElements
+        ? D(actions.removeAnalysisElement(applyResultElements))
```

Comment:
I'm gonna push back in this specific case.  Dropping the tertiary operators would require initializing (and naming) two new variables and moving their corresponding sub-expressions a bit from where they are actually used -- for what seems to be no purpose other than cutting down on colons and question marks.  Also considering how simple these particular cases are, I'm far from convinced that the suggested change would actually be an improvement.